source/adapters/level_zero/context.cpp

```diff
@@ -712,6 +712,11 @@ ur_result_t ur_context_handle_t_::getAvailableCommandList(
 
     for (auto ZeCommandListIt = ZeCommandListCache.begin();
          ZeCommandListIt != ZeCommandListCache.end(); ++ZeCommandListIt) {
+      // If this is an InOrder Queue, then only allow lists which are in order.
+      if (Queue->Device->useDriverInOrderLists() && Queue->isInOrderQueue() &&
+          !(ZeCommandListIt->second.flags & ZE_COMMAND_QUEUE_FLAG_IN_ORDER)) {
```

Comment:
Should be ZE_COMMAND_LIST_FLAG_IN_ORDER ? Instead of QUEUE?

Reply:
fixed in the latest update.